python/src/robyn/visualization/cluster_visualization.py

```diff
@@ -0,0 +1,111 @@
+# pyre-strict
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+from typing import Dict, Any
+import pandas as pd
+
+
+class ClusteringVisualizer:
```

Comment:
1. I see all methods here are static - which means it will take all data required in the method parameters only. If this is clustering visualization then it should take clustering data and any other data that is required by number of plot methods in initialization method only. 
2. Once we make this object oriented.. let's add method that plots all of them.. For example all_clustering_plots that plots all and returns them in list/dictionary.